### ISSUE TITLE
Cannot set screen resolution of simulator

### ISSUE THREAD

**Issue by mattbasta** [mozilla/r2d2b2g#272]
It's possible from the command line, but it cannot be done in the simulator.

**Comment by mykmelez** [comment]
We've talked about doing this, but we haven't prioritized it because Firefox OS's initial release will be on a single device with only one resolution.

But clearly that release will be followed by others on devices with different resolutions, and this will grow more important over time! So we should indeed provide a way for developers to specify a resolution for the Simulator that corresponds to one of the resolutions on commonly-available Firefox OS devices.

**Comment by mattbasta** [comment]
It's worthwhile to note that we will NOT be shipping with a single screen size: the Keon and Peak developer phones will have two different screen sizes/resolutions. It's also safe to assume that additional carriers will want to ship phones that have larger screen sizes and resolutions, as well. Whether this makes this feature more important is debatable, though.

**Comment by alimd** [comment]
What the exactly responsive solution for firefox os (not app) ?
Simulate dpi is not a solution ! we need a global and perfect solution for all ux/ui like responsive design
Is firefox os ui responsive ?

**Comment by nazar-pc** [comment]
I see, this bug is old, are there any plans for this feature?

**Comment by jryans** [comment]
Yes, we'd like to have such a feature. Follow [this bug](https://bugzil.la/948056) for updates.
